docs/GSMTC-Support-And-Popular-Apps.md

```diff
@@ -22,26 +22,27 @@ So, let's check the compatibility or support report.
 - 🟪 - Feature support is incomplete i.e. in-progress or being improved or expected to arrive soon or temporarily unavailable
 - 🟥 - Feature not supported
 
-| Application | Support | Play\Pause | Previous | Next | Thumbnail | Media Title | Media Artist | App Info | Shuffle | Repeat | Stop | Timeline information |
+| Application | Support | Play/Pause | Previous | Next | Thumbnail | Media Title | Media Artist | App Info | Shuffle | Repeat | Stop | Timeline information |
 | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
 | **Browsers** |
-| Chromium and its derivatives (such as **Chrome**, **New MS Edge**, **Opera**, **Brave**, **Vivaldi** and etc.) | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟥 | 🟥 | 🟥 | 🟥 |
+| Chromium and derivatives (Chrome, New MS Edge, Opera, Brave, Vivaldi, etc.) | 🟢 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟩 | 🟪 | 🟥 | 🟥 | 🟥 | 🟥 |
```

Comment:
Bruh! Why? My statement was just eloborative and bold.

Why tho?

Reply:
You see this big comment in table, you will see that it looks so ugly with a buttload of text in one of the cell in the table and nowhere else.

Reply:
What about adding a separate line to the doc for the same? It can contain this line and some more elaboration. It could be better

Reply:
Hmm, I agree on this one.

Reply:
agreed